generators/advanced/templates/root/_package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.1",
+  "description": "<%= description %>",
+  "author": {
+    "name": "<%= author %>",
+    "email": "<%= email %>"
+  },
+  "main": "index.js",
+  "scripts": {
+    "local": "SERVICE_ENV=local node index.js",
+    "start": "node index.js",
+    "infra-up": "docker-compose -f docker-compose.yml up -d",
+    "infra-down": "docker-compose -f docker-compose.yml down",
+    "sleep": "node test/helpers/sleep.js 20000",
+    "test": "npm run infra-up && npm run sleep && SERVICE_ENV=test mocha --reporter spec './test/**/*.test.js' && npm run infra-down",
+    "test-ci": "npm run infra-up && npm run sleep && SERVICE_ENV=test mocha --reporter mocha-junit-reporter && npm run infra-down",
+    "lint": "eslint .",
+    "manifest": "node_modules/make-manifest/bin/make-manifest",
+    "qa": "npm run lint && npm run test",
+    "precommit": "npm run lint",
```

Comment:
`precommit` and `prepush` seems redudant if we use also Husky

Reply:
right, new [issue](https://github.com/guidesmiths/generator-systemic/issues/50) for it since  is something that was already there prior to this PR.